### PR TITLE
Import fenix quarantine

### PR DESCRIPTION
**Pull request by mozilla-pontoon**
Import fenix quarantine

### CODE REVIEW

**Comment by mozilla-pontoon** [comment]
Run ID: 4810361291

## TOML file: firefox.toml

### File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml

**ID**: `juno_onboarding_default_browser_description_nimbus`
**Text**: `Firefox puts people over profits and defends your privacy by blocking cross-site trackers.\n\nLearn more in our privacy policy.`
**Errors:**
- Hard-coded brand `Firefox`. Use a variable instead.

**ID**: `juno_onboarding_default_browser_title_nimbus`
**Text**: `Make Firefox your go-to browser`
**Errors:**
- Hard-coded brand `Firefox`. Use a variable instead.

**Total errors not yet reported:** 2

**Review by Delphine — CHANGES_REQUESTED** [review]
@t-p-white: Pinging you since you landed the strings. Just a few quick fixes. Thank you!

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -283,10 +283,16 @@
     <!-- Title for set firefox as default browser screen.
         The first parameter is the name of the app defined in app_name (for example: Fenix) -->
     <string name="juno_onboarding_default_browser_title">Make %s your go-to browser</string>
+    <!-- Title for set firefox as default browser screen used by Nimbus. Nimbus does not support string placeholders.
+        Note: The word 'Firefox' should NOT be translated -->
```
[the remark this answers is not in the document]
Comment (Delphine) [reply mozilla-mobile/fenix/app/src/main/res/values/strings.xml:287]: Nit: should be "Title **to** set Firefox..." etc

**Review by Delphine — CHANGES_REQUESTED** [review]
(sorry for both pings, didn't realize I had already posted a review above)
@t-p-white: Pinging you here since I see you landed these two strings. There are just a few quick fixes needed, thank you!

(Note to self: I'll add exceptions for these two strings since "Firefox" isn't a placeholder)

**Comment by mozilla-pontoon** [comment]
Run ID: 4810361291

## TOML file: firefox.toml

### File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml

**ID**: `juno_onboarding_default_browser_description_nimbus`
**Text**: `Firefox puts people over profits and defends your privacy by blocking cross-site trackers.\n\nLearn more in our privacy policy.`
**Errors:**
- Hard-coded brand `Firefox`. Use a variable instead.

**ID**: `juno_onboarding_default_browser_title_nimbus`
**Text**: `Make Firefox your go-to browser`
**Errors:**
- Hard-coded brand `Firefox`. Use a variable instead.

**Total errors not yet reported:** 2

**Comment by flodolo** [comment]
Side note: the linter is behaving weirdly (not CCing the author) because, for some reason, those strings landed without an author.

Notice the icons [here](https://github.com/mozilla-mobile/firefox-android/commits/main/fenix/app/src/main/res/values/strings.xml) 

Screenshot 2023-04-27 alle 07 14 53

That might be also causing the comment to be repeated, but I'd need to verify that.

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (t-p-white) [reply mozilla-mobile/fenix/app/src/main/res/values/strings.xml:287]: Thanks for the feedback @Delphine 👍 I've added 'experiment' to the comment.

> Nit: should be "Title to set Firefox..." etc

I think **for** is correct here as it is describing a title for a 'type' of screen. Please see the lines 290-328 for more examples

**Comment by mozilla-pontoon** [comment]
Run ID: 4821434875

## TOML file: firefox.toml

### File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml

**ID**: `juno_onboarding_enable_notifications_description_nimbus`
**Text**: `Send tabs between devices, manage downloads, and get tips on getting the most out of Firefox.`
**Errors:**
- Hard-coded brand `Firefox`. Use a variable instead.

**ID**: `juno_onboarding_enable_notifications_title_nimbus`
**Text**: `Notifications help you do more with Firefox`
**Errors:**
- Hard-coded brand `Firefox`. Use a variable instead.

**Total errors not yet reported:** 2

**Review by Delphine — APPROVED** [review]

**Comment by Delphine** [comment]
Ah just realized linter is still complaining because of the two new strings with "Firefox" in them. I'll add those to exceptions as well now

**Pull request merged**